### DISCUSSION TITLE
Can not import Scratchpad

### DISCUSSION THREAD

**Discussion by ivanuricardo** [qtile/qtile#4498] [General]
Hello! 
I am having trouble with the recent update to python 3.11. I had the same issue as [issue 4249](https://github.com/qtile/qtile/issues/4249) which was fixed when I swapped the lines
 
```
from libqtile.config import Key
from libqtile.lazy import lazy
```

and I now get a different error message when I run 
`python3 ~/.config/qtile/config.py` which is

Traceback (most recent call last):
  File "/home/ivan/.config/qtile/config.py", line 2, in <module>
    from modules.groups import groups
  File "/home/ivan/.config/qtile/modules/groups.py", line 1, in <module>
    from libqtile.config import Scratchpad, DropDown, Key, Group
ImportError: cannot import name 'Scratchpad' from 'libqtile.config' (/usr/lib/python3.11/site-packages/libqtile/config.py)

Any ideas on how to fix this? Thanks!

**Comment by elParaguayo** [comment]
Try `ScatchPad` (capital `P`).
